README.md

```diff
@@ -736,6 +742,154 @@ No errors detected
 
 ```
 
+## `mcci-catena-provision-sigfox.py`

```

Comment:
I'm uncomfortable about putting all this together in a single repo (lorawan, helium, etc). We can't easily maintain version tags, isuses, etc. We need to think about splitting this up into a library plus programs for specific networks.